Project/feature-u-soundboard/src/feature_config.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Dit is generated toch? Misschien handig om dan dit toe te voegen aan de gitignore.